### PR TITLE
Improve phrasing of constants in GDScript basics

### PR DESCRIPTION
**Pull request by NathanLovato**
Based on a user's request on the Discord server's documentation channel.

### CODE REVIEW

File: getting_started/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -654,8 +654,11 @@ the scene tree::
 Constants
 ~~~~~~~~~
 
-Constants are similar to variables, but must be constants or constant
-expressions and must be assigned on initialization.
+Constants are values you cannot change when the game is running. Using the
+``const`` keyword allows you to give a constant value a name. Trying to assign a
+value to a constant will give you an error.
```
Comment (Calinou) [inline getting_started/scripting/gdscript/gdscript_basics.rst:659]:
```suggestion
value to a constant after it's declared will give you an error.
```

File: getting_started/scripting/gdscript/gdscript_basics.rst [hunk 2]
```diff
@@ -654,8 +654,11 @@ the scene tree::
 Constants
 ~~~~~~~~~
 
-Constants are similar to variables, but must be constants or constant
-expressions and must be assigned on initialization.
+Constants are values you cannot change when the game is running. Using the
```
Comment (Calinou) [inline getting_started/scripting/gdscript/gdscript_basics.rst:657]:
```suggestion
Constants are values you cannot change when the game is running.
Their value must be known at compile-time. Using the
```

**Review by Calinou — COMMENTED** [review]
I'm not sure if the new formulation is actually an upgrade, as it no longer mentions the requirement that constants should have a value known at compile-time. (This excludes things such as random numbers or OS-specific values.)

My suggestion might not be the best way to phrase it, but we need to change something before this can be merged.

**Comment by NathanLovato** [comment]
Thanks, I added your suggestions to the PR

**Comment by Calinou** [comment]
Thanks :slightly_smiling_face:

That said, https://github.com/godotengine/godot-docs/pull/3623 is also overhauling that page. `const` can now be used at local scope in `master`, but this isn't possible in 3.2.x.

**Comment by NathanLovato** [comment]
I just checked the changes in #3623 and they don't conflict with these ones. Vnen didn't edit the first paragraph below the Constants heading.

**Pull request merged**